packages/integration-karma/test/misc/cross-domain-iframe/index.spec.js

```diff
@@ -16,7 +16,7 @@ it('should not throw when unwrapping contentWindow', () => {
 describe('HTMLIFrameElement.contentWindow patching', () => {
     let iframe, sameOriginFrame;
 
-    beforeAll(() => {
+    beforeEach(() => {
```

Comment:
This is going to add overhead because the test setup is going to run for each case. IMO, the test should clean up in an afterAll() rather than removing the ability to use beforeEach()

Reply:
The problem is that we would have to make each test individually responsible for cleaning up its own DOM nodes added in `beforeAll()`. This introduces risk of developer error and adds a lot of extra cleanup code.

I'll benchmark this and see how much overhead it adds exactly.

Reply:
I ran [a benchmark](https://docs.google.com/spreadsheets/d/1iS0MWge7l3l66i7Sl1C4WmA64X5npSGSNsZv8Iv7YRc/edit?usp=sharing) of `yarn test` (in Chrome) and, based on the median of 5 runs, the test time increased from 11.8s to 12.055s, or 255ms (2.16%). I would argue that this slowdown is acceptable.

Reply:
Sounds like a good tradeoff. Can you please add documentation in a README to warn future developers to not use beforeAll() in the karma suite? TIA

Reply:
If a developer does that, then the test should fail because the DOM is now dirtied at the end of the test. But sure, I will add something to the README to note this.